config/production.json

```diff
@@ -3,10 +3,32 @@
     "declarationsPath": "../ota-services-all/declarations",
     "documentTypesPath": "../ota-services-all/document-types.json"
   },
-  "history": {
-    "snapshotsPath": "../ota-snapshots",
-    "versionsPath": "../ota-versions",
-    "publish": true
+  "recorder": {
+    "versions": {
+      "storage": {
+        "git": {
+          "path": "../ota-versions",
+          "publish": true,
+          "prefixMessageToSnapshotId": "This version was recorded after filtering snapshot https://github.com/ambanum/OpenTermsArchive-snapshots/commit/",
+          "author": {
+            "name": "Open Terms Archive Bot",
```

Comment:
I would keep OTA Bot as it is smaller on commit lines
![Screen Shot 2021-12-14 at 14 08 03](https://user-images.githubusercontent.com/4191809/145977663-0f3c2792-91f9-4d2c-898e-268843db8e1a.png)

Reply:
Since Git and its metadata is part of our user-visible UI, I prefer that we expand the name:

<img width="458" alt="Capture d’écran 2021-12-14 à 09 44 21" src="https://user-images.githubusercontent.com/222463/146000663-d2b23b2d-9667-4b39-aa5c-93d8e2a4a356.png">

Reply:
Thanks to this being in config, maybe we can have the best of both world with using `OTA Bot` in the dev config and `Open Terms Archive Bot` in production 🙂